app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/notes/NotesContracts.kt

```diff
@@ -0,0 +1,11 @@
+package org.dhis2.usescases.teiDashboard.dashboardfragments.notes
+
+import org.dhis2.usescases.general.AbstractActivityContracts
+import org.hisp.dhis.android.core.note.Note
+
+class NotesContracts {
+
+    interface View : AbstractActivityContracts.View {
+        fun swapNotes(noteModules: List<Note>)
+    }
+}
```

Comment:
We are not using the contract class. This should be the interface only. - `NotesVies`